jekyll/_cci2/concepts.md

```diff
@@ -7,61 +7,95 @@ categories: [getting-started]
 site.baseurl }}/2.0/gh-bb-integration/) to view or follow associated CircleCI
-projects.  Users may not view project data that is stored in environment variables.
+projects. Users may not view project data that is stored in environment variables.
 
-## Steps
+## Pipelines
 
-Steps are actions that need to be taken to perform your job. Steps are usually a collection of executable commands. For example, the `checkout` step checks out the source code for a job over SSH. Then, the `run` step executes the `make test` command using a non-login shell by default.
+A CircleCI pipeline is the full set of processes you run when you trigger work on your projects. Pipelines encompass your workflows, which in turn coordinate your jobs. This is all defined in your project [configuration file](#config).
 
-```yaml
-#...
-    steps:
-      - checkout # Special step to checkout your source code
-      - run: # Run step to execute commands, see
-      # circleci.com/docs/2.0/configuration-reference/#run
-          name: Running tests
-          command: make test # executable command run in
-          # non-login shell with /bin/bash -eo pipefail option
-          # by default.
-#...          
+Pipelines offer the following benefits:
+
+{% include snippets/pipelines-benefits.adoc %}
+
+## Orbs
+Orbs are reusable snippets of code that help automate repeated processes, speed up project setup, and make it easy to integrate with third-party tools. See [Using Orbs]({{ site.baseurl }}/2.0/using-orbs/) for details about how to use orbs in your config and an introduction to orb design. Visit the [Orbs Registry](https://circleci.com/orbs/registry/) to search for orbs to help simplify your config.
+
+## Config
+
+CircleCI believes in *configuration as code*. The entire pipeline process, from build to deploy, is orchestrated through a single file called `config.yml`.  The `config.yml` file is located in a folder called `.circleci` at the root of your project. CircleCI uses the YAML syntax for config, see the [Writing YAML]({{ site.baseurl }}/2.0/writing-yaml/) document for basics.
+
+```sh
+├── .circleci
+│   ├── config.yml
+├── README
+└── all-your-projects-other-files-and-folders
 ```
 
-## Image
+`config.yml` is a powerful yaml file that defines the entire pipeline for your project. For a full overview of the various keys that can be used see the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/). 
+
+Put very simply, you will define [workflows](#workflows) that orchestrate a series of [jobs](#jobs). Each job will contain a number of [steps](#steps) to run commands and shell scripts to do the work required for your project. Each job runs in an independent [executor](#executors-and-images) (container or virtual machine). [Caches](#caches-workspaces-and-artifacts) are available to optimize and speed up pipelines, and [workspaces/artifacts](#caches-workspaces-and-artifacts) can be used to share data across your pipeline.
```

Comment:
I'm wondering if this is worth breaking down into an un/ordered list? Thoughts?

Reply:
Yes, I think that would work well. I'll try it.